src/js/modules/infragistics.util.js

```diff
@@ -1530,6 +1530,20 @@
 		Date.prototype.toISOString = Date.prototype.toJSON;
 	}
 
+	// polyfill for IE11+. ChildNode.remove() is not supported by IE11+.
+	function remove() {
```

Comment:
@Aleksandyr My only concern is that the function name is very generic. How about `removePolyfillIE`

Reply:
@mpavlinov I agree. I'm going to change the name with the proposed one.